ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.hbs

```diff
@@ -12,18 +12,26 @@
 
   {{#each @destination.formFieldGroups as |fieldGroup|}}
     {{#each-in fieldGroup as |group fields|}}
-      {{#if (and (eq group "Credentials") (not @destination.isNew))}}
+      {{! h2 titles Credentials and Configuration for FormFieldGroups to be displayed in both create and edit }}
+      {{#if (eq group "Credentials")}}
```

Comment:
As a suggestion, if the `config` form field group changed to the section header title of `Advanced configuration` the `::display` and `::body` template logic could be reused a bit, to something like the following (which would also require a getter for the subtext)

```hbs
{{#if (not-eq group "default")}}
  <hr class="has-top-margin-xl has-bottom-margin-l has-background-gray-200" />
  <Hds::Text::Display @tag="h2" @size="400" @weight="bold">{{group}}</Hds::Text::Display>
  <Hds::Text::Body @tag="p" @size="100" @color="faint" class="has-bottom-margin-m">
    {{this.groupSubtext}}
  </Hds::Text::Body>
{{/if}}